docs/source/transaction_family_specifications/supplychain_transaction_family.rst

```diff
@@ -0,0 +1,317 @@
+***************************************
+Supply Chain Transaction Family
+***************************************
+
+Overview
+=========
+
+The Supplychain Transaction Processor(SCTP) provides a mechanism for
+storing records of asset ownership. The SCTP tracks records, which correlate
+to a real world item. Every Record has an owner and a custodaian, which can
+be the same Agent. The owner is the Agent the legally owns the good. The
+custodian is the Agent that is in possesion of the good, which may be a
+warehousing service, transportation service, etc. Agents are actors in the
+system that are can manupulate the Records by updating them or transfering
+their role to other Agents. Agents are identified by their public key and
+Authenticated by the signature on the transactions they submit. Transfer of
+roles(Owner, Custodian) betweeen Agents is done via an Application process,
+Where the Agent wishing to assume the role Applies for the new role and the
+Agent in the role currently can either accept or reject that applicaiton.
+
+This is an proof of concept Transaction Processor that is actively being
+developed. As such it has not been thoughly tested and there will be
+protentially breaking changes made to it.
+
+State
+=====
+
+This section describes in detail how SCTP objects are stored and addressed in
+the global state. All objects in state are stored as encoded protobufs. Each
+type of object below is stored in containers at nodes in the Merkle Trie. The
+address of the Merkle nodes is detailed below in the addressing section. The entry at the nodes is a container object that allows for multiple items to be stored at that node if there is an address collision.
```

Comment:
Need to describe the storage format for collisions (in the state section), lexicographically sorted by natural keys, etc.